src/main/java/net/mcreator/ui/gradle/GradleErrorDialogs.java

```diff
@@ -63,24 +64,22 @@ else if (errorCode == GradleErrorCodes.JAVA_INVALID_VERSION)
 
 		return errorCode;
 	}
-	
-	private static final String MESSAGE_TITLE = "Gradle task failed with error";
-	
+
+	private static final String MESSAGE_TITLE = L10N.t("gradle.errors.title");
+
 	private static String applyAppendx(String msg, int errorCode) {
-		String appendx = "<small><br><br><font color=gray>ERROR CODE: " + GradleErrorCodes.toString(errorCode) + " [" + errorCode + "]";
+		String appendx = "<small><br><br><font color=gray>" + L10N.t("gradle.errors.error") + GradleErrorCodes
+				.toString(errorCode) + " [" + errorCode + "]";
 		return msg + appendx;
 	}
 
 	private static void showGradleCacheDataErrorDialog(MCreator whereToShow, int errorCode) {
 		if (PreferencesManager.PREFERENCES.gradle.offline) {
 			showGradleCacheOutdatedDialogOfflineMode(whereToShow, errorCode);
 		} else {
-			String msg = "<html><big>Gradle caches are corrupted!</big><br>"
-					+ "<br>It seems that the cached dependencies of Gradle got corrupted or are outdated.<br>"
-					+ "Cache needs to be cleared and re-downloaded. You can do this by pressing the button below.<br>"
-					+ "<br>After you clear cache, the first Gradle build could take a but longer than usual.";
+			String msg = L10N.t("gradle.errors.cache_corrupted");
 
-			String[] options = { "Clear Gradle caches", "Do nothing" };
+			String[] options = { L10N.t("gradle.errors.clear_caches"), L10N.t("gradle.errors.do_nothing") };
```

Comment:
These keys do not match with texts file

Reply:
Please double check the keys in case there are more like this one

Reply:
These ones happened because I decided to change the key name after, but I forgot to change them back in the texts.properties.